src/primitives/block.h

```diff
@@ -183,7 +183,7 @@ class CBlock : public CBlockHeader
 
     // return the sc txs commitment calculated as described in zendoo paper. It is based on contribution from
```

Comment:
Maybe the description of this method can be adapted like:

```
// Builds the sc txs commitment calculated as described in zendoo paper. It is based on contribution from
// sidechains-related txes and certificates contained in this block. Returns the building result
```

Reply:
Reworded in e6db47d33